README.md

```diff
@@ -1,96 +1,31 @@
+[![MELPA](https://melpa.org/packages/rust-mode-badge.svg)](https://melpa.org/#/rust-mode)
+
 `rust-mode`: A major Emacs mode for editing Rust source code
 ============================================================
 
-`rust-mode` makes editing [Rust](http://rust-lang.org) code with Emacs
-enjoyable.  `rust-mode` requires Emacs 24 or later.
-
-## Installation
-
-### Manual Installation
-
-To install manually, check out this repository and add this to your
-`.emacs` file:
-
-```lisp
```

Comment:
I think that the install instructions are useful. Note that use-package does come as a built-in only for recent versions.